into/backends/aws.py

```diff
@@ -235,3 +237,21 @@ def remote_text_to_s3_text(a, b, **kwargs):
 @append.register(HDFS(TextFile), S3(TextFile))
 def other_remote_text_to_s3_text(a, b, **kwargs):
     raise MDNotImplementedError()
+
+
+if sys.platform != 'win32':
```

Comment:
ok, yep. you're right. you can use paramiko to connect to a remote host on windows. i guess i shouldn't be surprised :)